Exiled.API/Features/Core/GameEntity.cs

```diff
@@ -44,11 +44,6 @@ protected GameEntity(GameObject gameObject)
             // List.Add(this);
         }
 
-        /// <summary>
-        /// Finalizes an instance of the <see cref="GameEntity"/> class.
-        /// </summary>
-        ~GameEntity() => List.Remove(this);
-
```

Comment:
Why is this being removed?

Reply:
Never call, [same reason](https://github.com/Exiled-Team/EXILED/pull/2764#discussion_r1712646105).